.github/workflows/test.yaml

```diff
@@ -62,14 +68,36 @@ jobs:
           ~/.pyenv
           ~/Library/Caches/pip
           ~/Library/Caches/pipenv
-          ~/Library/Caches/Homebrew
-        key: ${{ hashFiles('tools/mac_setup.sh') }}
+          /usr/local/Cellar
+          ~/github_brew_cache_entries.txt
+        key: macos-deps-${{ hashFiles('tools/mac_setup.sh') }}
+        restore-keys: macos-deps-
+    - name: Brew link restored dependencies
+      if: steps.dependency-cache.outputs.cache-hit == 'true'
```

Comment:
is this if necessary? should do nothing if ~/github_brew_cache_entries.txt doesn't exist